composer.json

```diff
@@ -15,6 +15,10 @@
         {
             "name": "Carsten Brandt",
             "email": "mail@cebe.cc"
+        },
+        {
+            "name": "gengxiankun",
+            "email": "gengxiankun@126.com"
```

Comment:
Although your contribution is appreciated, this doesn't make you eligible for being this lib author (or we'd have to add 135 other names here). Please revert.